pkg/koordlet/util/system/cgroup.go

```diff
@@ -207,6 +207,29 @@ func ReadCgroupAndParseUint64(parentDir string, r Resource) (uint64, error) {
 	return v, nil
 }
 
+func ReadCgroupAndParseInt32Slice(parentDir string, r Resource) ([]int32, error) {
```

Comment:
more comments needed as example for this func

Reply:
done